### ISSUE TITLE
 Generator "nexus-prisma" failed with prisma version `5.9~`

### ISSUE THREAD

**Issue by POPPIN-FUMI** [prisma/prisma#23052]
### Bug description

`npx prisma generate` fails when I update prisma version from `5.8.1` to `5.9~`

ok
package.json
```
"dependencies": {
    "@apollo/server": "4.10.0",
    "@jcm/nexus-plugin-relay-global-id": "0.2.0",
    "@jcm/nexus-plugin-relay-node-interface": "0.2.0",
    "@prisma/client": "5.8.1",
    "@skeet-framework/utils": "1.2.4",
    "firebase-admin": "12.0.0",
    "graphql": "16.8.1",
    "graphql-depth-limit": "1.1.0",
    "graphql-middleware": "6.1.35",
    "graphql-query-complexity": "0.12.0",
    "graphql-relay": "0.10.0",
    "graphql-shield": "7.6.5",
    "nexus": "1.3.0",
    "nexus-prisma": "2.0.3",
    "prisma": "5.8.1"
  },
 ```

error
```
"dependencies": {
    "@apollo/server": "4.10.0",
    "@jcm/nexus-plugin-relay-global-id": "0.2.0",
    "@jcm/nexus-plugin-relay-node-interface": "0.2.0",
    "@prisma/client": "5.9.1",
    "@skeet-framework/utils": "1.2.4",
    "firebase-admin": "12.0.0",
    "graphql": "16.8.1",
    "graphql-depth-limit": "1.1.0",
    "graphql-middleware": "6.1.35",
    "graphql-query-complexity": "0.12.0",
    "graphql-relay": "0.10.0",
    "graphql-shield": "7.6.5",
    "nexus": "1.3.0",
    "nexus-prisma": "2.0.3",
    "prisma": "5.9.1"
  },
 ```

I test `~5.8.1` was okay.
I get the title's error since the version `5.9.0`

Thank you!

### How to reproduce

package.json
```
"dependencies": {
    "@apollo/server": "4.10.0",
    "@jcm/nexus-plugin-relay-global-id": "0.2.0",
    "@jcm/nexus-plugin-relay-node-interface": "0.2.0",
    "@prisma/client": "5.9.1",
    "@skeet-framework/utils": "1.2.4",
    "firebase-admin": "12.0.0",
    "graphql": "16.8.1",
    "graphql-depth-limit": "1.1.0",
    "graphql-middleware": "6.1.35",
    "graphql-query-complexity": "0.12.0",
    "graphql-relay": "0.10.0",
    "graphql-shield": "7.6.5",
    "nexus": "1.3.0",
    "nexus-prisma": "2.0.3",
    "prisma": "5.9.1"
  },
 ```

`npx prisma generate`

```
Error: Generator "nexus-prisma" failed:

node:internal/modules/cjs/loader:598
      throw e;
      ^
Error: Cannot find module '/Users/fumi/Dev/ts/skeet-framework/templates/skeet-graphql/graphql/node_modules/@prisma/client/generator-build'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1181:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1169:15)
    at resolveExports (node:internal/modules/cjs/loader:591:14)
    at Module._findPath (node:internal/modules/cjs/loader:668:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1130:27)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/Users/fumi/Dev/ts/skeet-framework/templates/skeet-graphql/graphql/node_modules/nexus-prisma/dist-cjs/lib/prisma-utils/externalToInternalDMMF.js:6:52)
    at Module._compile (node:internal/modules/cjs/loader:1376:14) {
  code: 'MODULE_NOT_FOUND',
  path: '/Users/fumi/Dev/ts/skeet-framework/templates/skeet-graphql/graphql/node_modules/@prisma/client/package.json'
}
Node.js v20.11.0
```

### Prisma information

```prisma
generator client {
  provider = "prisma-client-js"
}

generator nexusPrisma {
  provider = "nexus-prisma"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

```bash
npx prisma generate
```

### Environment & setup

- OS: macOS
- Database: PostgreSQL
- Node.js version: 20.11.0

### Prisma Version

```
5.9.1
```

**Comment by Kuhave** [comment]
duplicate #22927( please check it 😉 )

I've added a PR #23015, and merged.

Perhaps a fix is available in version 5.10.0.

**Comment by Jolg42** [comment]
I'll close as a duplicate of https://github.com/prisma/prisma/issues/22927

@POPPIN-FUMI you can try Prisma version `5.10.0-dev.48` for the fix.

Wait for `5.10.0` release for an official version to use.
